syncd/Syncd.cpp

```diff
@@ -2122,7 +2122,12 @@ void Syncd::processFlexCounterEvent( // TODO must be moved to go via ASIC channe
 
     sai_object_id_t rid;
 
-    if (!m_translator->tryTranslateVidToRid(vid, rid))
+    // It's possible the object was removed before the corresponding counter is removed
+    // even if orchagent removed counter first and then the object itself because
+    // they are handled in different tables in syncd
+    // In such case, error message will be reported in tryTranslateVidToRid
+    // To avoid the error, the function is called for SET only
+    if (op == SET_COMMAND && !m_translator->tryTranslateVidToRid(vid, rid))
```

Comment:
comment is correct, but please remove this operation, this issue was fixed here: https://github.com/Azure/sonic-sairedis/pull/1048/files but not merged yet, if you want you can leave comment why trytranslate is used

Reply:
Thanks. Then I'll close this PR and let's discuss there.